### DISCUSSION TITLE
windows10 use vrep

### DISCUSSION THREAD

**Discussion by wobushixiaozhu** [jhu-dvrk/sawIntuitiveResearchKit#208] [Q&A] [answered]
I want to run the DVRK model with vrep via windows10, is there a configured model?

**Comment by pkazanzides** [comment, marked as answer]
As noted here, https://github.com/jhu-dvrk/sawIntuitiveResearchKit/wiki#software-ecosystem, there is a V-Rep based simulator developed at University of Naples.  The link to that project is https://github.com/unina-icaros/dvrk-vrep.
